.github/workflows/tests.yml

```
@@ -0,0 +1,56 @@
+name: Tests
+
+on:
+  pull_request:
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [8.1, 8.0, 7.4]
+        wp_version: ["latest"]
+        can_fail: [false]
+        multisite: [0,1]
```

What's the benefit of including these single-item lists in the matrix? Would environment variables via a top-level `env` map do the trick?

Also: it doesn't look like `can_fail` appears anywhere else in this pull request, so unless there's a good reason to keep it, I'd recommend removing it.

```suggestion
env:
  WP_VERSION: latest

jobs:
  phpunit:
    runs-on: ubuntu-latest
    strategy:
      fail-fast: true
      matrix:
        php: [8.1, 8.0, 7.4]
        multisite: [0,1]
```

---

I think they would work as environment variables, but I'd ask in return: What's the benefit of making them so? They seem to work right now as lists of one item, and as demonstrated by the suggested changes filed under "if you make them env vars," it's a non-zero-effort task to switch them back to being in the matrix if the list ever grows beyond one.

`can_fail` appears here: https://github.com/alleyinteractive/wp-find-one/blob/574f5851cc4ef58a9ea78655fd09b602546d39f3/.github/workflows/tests.yml#L24